azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/database/DatabaseService.java

```diff
@@ -272,6 +273,8 @@ public void pgRestore(
       }
     }
     localProcessLauncher.getOutputStream().flush();
+    localProcessLauncher.getOutputStream().close();
+    localProcessLauncher.waitForTerminate();
```

Comment:
Is there any kind of timeout that can be added here? i.e., how long are we willing to wait for this process to die before giving up?

Reply:
Not here but I believe there is an overall pod time limit asserted by the wrapping task